src/server/apis/dash-data.ts

```diff
@@ -148,6 +149,37 @@ export class DashData {
           reviewedPr.events.push({type: 'MyReviewEvent', review: myReview});
         }
 
+        // Check if there are new commits to the pull request.
+        if (pr.commits.nodes) {
+          const newCommits = pr.commits.nodes.filter(
```

Comment:
Do this filter in the loop below so that you don't loop twice and don't parse the `pushedDate` twice? Fine if you prefer this, but seems simpler to me to do one loop.

Reply:
Fair point. Thanks, done.